frontend/test/metabase/scenarios/permissions/application-permissions.cy.spec.js

```diff
@@ -100,7 +100,9 @@ describeEE("scenarios > admin > permissions > application", () => {
         cy.signInAsNormalUser();
       });
 
-      it("allows accessing tools, audit, and troubleshooting for non-admins", () => {
+      // Adding this test to quarantine. When it was failing was making all the subsequents to fail.
+      // More details can be found on the Thread https://metaboat.slack.com/archives/CKZEMT1MJ/p1649272824618149
```

Comment:
Note that a summary in a GitHub issue as the reference will be more useful (than some hard-to-discover-later Slack thread).

Reply:
I've added the resume on GitHub (with the info we have right now). I've added the slack thread because things are still being discussed there. So, for history porposal can be more helpful.